images/kor/main.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
```

Comment:
you should create a config module here to reference the ./config folder you create for managing the image configuration.